api/src/routes/protected/certificate.ts

```diff
@@ -268,7 +268,10 @@ export const protectedCertificateRoutes: FastifyPluginCallbackTypebox = (
     async (req, reply) => {
       const { certSlug } = req.body;
 
+      const child = fastify.log.child({ req });
```

Comment:
```suggestion
      const logger = req.log.child({ req });
```
This way the `reqId` automatically gets attached to the log. Also "logger" is a bit more descriptive.

Reply:
Why do we need both `req.` and `{ req }`?

Reply:
I've not been able to figure out a way to create a child logger with that `req` passed to it.

I'll give it another go, but I'm not sure it's possible.